local FauxPhysics = require(script:WaitForChild("FauxPhysics"))
local DragController = require(script:WaitForChild("DragController"))
local CookingController = require(script:WaitForChild("CookingController"))
local SprintController = require(script:WaitForChild("SprintController"))
local ItemLifetime = require(script:WaitForChild("ItemLifetime"))
local DropperUI = require(script:WaitForChild("DropperUI"))

FauxPhysics.Start()
DragController.Start()
CookingController.Start()
SprintController.Start()
ItemLifetime.Start()
DropperUI.Start()
